cg/models/taxprofiler/taxprofiler.py

```diff
@@ -40,7 +41,7 @@ class TaxprofilerParameters(WorkflowParameters):
     centrifuge_save_reads: bool = True
     run_krona: bool = True
     run_profile_standardisation: bool = True
-    clusterOptions: str = Field(..., alias="cluster_options")
+    clusterOptions: str | None = Field(str | None, alias="cluster_options")
```

Comment:
```suggestion
    clusterOptions: str | None = Field(None, alias="cluster_options")
```
I thought the first entry in "Field" is the default value but if not, please resolve. Why are we making this optional though? It seems required in the previous commit

Reply:
👍

Reply:
Fixed